### DISCUSSION TITLE
Total Noob, how do I set it up

### DISCUSSION THREAD

**Discussion by Mokerak** [openzfsonwindows/openzfs#309] [Q&A] [answered]
I have ran the installer for windows, now what?

I see the install in Program Files, theres executables such as zfs.exe, zpool.exe, etc. 

I downloaded the debug.exe

Do I have to run in linux and create a zfs partition, and windows will recognize and be able to use it? 
Do I need to compile a release executable using Inno Setup Compiler?

I'm sort of a pro at windows, and a big noob at Linux

**Comment by EchterAgo** [comment]
There is a Windows specific README here: https://github.com/openzfsonwindows/openzfs/tree/windows/module/os/windows#installing-a-binary-release

The commands that you'd use are mostly the same as on Linux.

**Reply by EchterAgo** [reply]
Note that the current RC5 binary has some known issues, hopefully RC6 will be better once released.

**Comment by lundman** [comment, marked as answer]
Basically, install, then start CMD.exe as Administrator, and type "zpool status".

**Comment by Mokerak** [comment]
THANKS!!!

![Capture](https://github.com/openzfsonwindows/openzfs/assets/34557152/aab0e0db-ec44-4b47-a1a1-910a8cc12036)

**Reply by Mokerak** [reply]
Before with Windows Storage Spaces ---- After with ZFS

4-4tb hard drives raidz

**Reply by EchterAgo** [reply]
Which one of these is ZFS? Storage Spaces can be extremely slow with parity unless you're very careful with the settings. My frustration with Storage Spaces and ReFS is what made me consider the ZFS Windows port.

**Reply by Mokerak** [reply]
The one on the right is ZFS

**Reply by EchterAgo** [reply]
@Mokerak FYI this is what I get when trying CDM on a 9 * 18TB raidz1 pool:

![image](https://github.com/openzfsonwindows/openzfs/assets/922233/65112a85-3092-46b0-be30-64190ce99e00)

Note that I'm using compression and a 1MB recordsize.

**Reply by EchterAgo** [reply]
Note that this is also using a larger size than can fit in RAM so it can't just RAM cache everything. IIRC CrystalDiskMark should normally disable caching, but I'm not sure if that is actually respected by the ZFS Windows port.

**Reply by Mokerak** [reply]
I don't think so, I did a test with 64GB of data and because ZFS couldn't cache it all, the results were much more reasonable even for a highly efficient filesystem

**Comment by guenther-alka** [comment]
If you want to do performance tests, disable compress or you test compress quality not pool quality
I would not expect CDM to disable ZFS caches but you can disable firstlevel (Arc) or secondlevel (L2Arc) read caches in ZFS. But this disables metadata cache as well so expect very low values unless you use fast NVMe. ZFS needs the caches to be fast despite its security advantage over older filesystems. 

btw
ZFS does not caches files but ZFS datablocks on a read last/ read most base.

If you want to simplify ZFS management on Windows, you can try my first preview of napp-it, 
a Windows port of my web-gui for ZFS on Solaris . No installation needed, just download and run.
